frontend/lib/src/password_reset_page.dart

```diff
@@ -0,0 +1,103 @@
+import 'package:flutter/material.dart';
+import 'package:frontend/consts/colors.dart';
+import 'package:frontend/src/components/auth/auth_header.dart';
+import 'package:frontend/src/components/auth/input_field.dart';
+import 'package:frontend/src/password_reset_confirm_page.dart';
+import 'package:get/get.dart';
+
+class PasswordResetPage extends StatelessWidget {
+  const PasswordResetPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: passwordResetBackgroundColor,
```

Comment:
Prefer to use `ThemeData` instead of manually adding a constant color.

You can find the Theme in main.dart

To use, just use `Theme.of(context).backgroundColor`